src/configuration/services/mysql.md

```diff
@@ -164,6 +166,26 @@ db:
 
 The above code will increase the maximum allowed packet size (the size of a query or response) to 64 MB.  However, increasing the size of the maximum packet will also automatically decrease the `max_connections` value.  The number of connections allowed will depend on the packet size and the memory available to the service.  In most cases leaving this value at the default is recommended.
 
+## Character encoding
+
+For services created prior to February 2020, the default character set and collation is `latin1`, which is the default in most MySQL/MariaDB.
```

Comment:
Can we also add something about how to change this on existing databases/tables if it is set incorrectly?

`ALTER DATABASE db_name CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;`
`ALTER TABLE table_name CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;`

That'll avoid answering this question in support ;-)

Reply:
Eh, I was hoping to delegate that to the MySQL docs, hence the link further on.  I suppose we can include some stock commands though, if no one has an objection.